src/Provider/GoogleMaps/GoogleMaps.php

```diff
@@ -264,9 +264,42 @@ private function fetchUrl(string $url, string $locale = null, int $limit, string
             if (isset($result->formatted_address)) {
                 $address = $address->withFormattedAddress($result->formatted_address);
             }
+            if ($builder->hasValue('street_address')) {
+                $address = $address->withStreetAddress($builder->getValue('street_address'));
+            }
```

Comment:
Since all of the address setters allow null, can't we write this code more simple? Without the extra `if`?

`$address = $address->withStreetAddress($builder->getValue('street_address'));`